### ISSUE TITLE
Documentation Organization

### ISSUE THREAD

**Issue by cbizon** [clingen-data-model/clingen-interpretation#165]
The current documentation organization is pretty good, but there are still a couple of questions that I would like opinions on.

1) When we look at the entities under "Core" we see this:

CriterionAssessment
DomainEntity
EvidenceLine
GeneticCondition
Statement
VariantInterpretation

I think that GeneticCondition looks kind of out of place here.  It's more like the things under DomainEntity, except that it doesn't actually descend from DomainEntity.

2) Are we going to include Contextual/Canonical Allele (probably not).  How about haplotype and genotype (probably).  Where will they go?

**Comment by larrybabb** [comment]
good call out.
1.  I agree with moving GeneticCondition under DomainEntities.
2. I do think we should should have Genotype/Haplotype and potentially Contextual and CanonicalAllele, but we need to finalize our discussion related to these types first.  Let's revisit after our next meeting on these types.

**Comment by cbizon** [comment]
Then should we change the header on "domain entities" to something else, since not everything under that heading will actually be a domain entity?

**Comment by larrybabb** [comment]
That's fine. I think "domainEntities" is something we pulled out of the air to represent the root "Entity" type.  I don't think we want to formally show our internal DomainEntity superclass.  I suppose we could align it with that [InformationContentEntity](https://github.com/monarch-initiative/SEPIO-ontology/wiki/Information-Content-Entity-(aka-Information)) (or whatever it is called) from the ontological world.  Or it's supertype "Generically Dependent Continuant".  See this page for reference

**Comment by cbizon** [comment]
Well, remember that DomainEntity has a few jobs, and that it is basically there as the replacement for CodeableConcept.  So it doesn't exactly line up with "information" or whatever, because it's really something we came up with to  handle how we wanted to be able to put user labels on things and so on. 

So I guess I think that we do want to have a page for DomainEntity (and Statement, which is also a superclass), because it's helpful for only explaining some of the usage in one place.

**Comment by larrybabb** [comment]
hmmm...
we should definitely discuss this today on the call.
I have "utilized" the sheet structure for DomainEntity and DomainEntityAttribute for a number of things that may not qualify for CodeableConcept or things that possibly should not inherit the "UserLabel" and any other DomainEntity characteristics.  

I thought that DomainEntity was more of a "if it isn't our clingen-dmwg defined entity then put it here". The UserLabel is for a "special" class that allows these DomainEntities to be wrapped, re-labeled and custom created by senders.

But I'm thinking I have it backwards or we may not have enough buckets to distinguish this nuance.

We'll talk.

**Comment by cbizon** [comment]
Yeah,  we basically decided that anything from a ValueSet would be one of these things, right?  So that means that e.g. we now allow users to put labels on Genes.  And at the same time, we use this superclass for anything that doesn't come from a value set, like Family.

It's not necessarily bad, but we can certainly discuss it further.  But we need to drive towards a resolution in the discussion.

**Comment by larrybabb** [comment]
agreed. It's probably just my confusion.

**Comment by cbizon** [comment]
Also right now, haplotype is under EvidenceStatment, and we need IRIs for things in the genotype/haplotype stuff.
